src/tiled/scriptfileinfo.cpp

```diff
@@ -242,3 +242,74 @@ QString ScriptFileInfo::symLinkTarget(const QString &file) const
 {
     return QFileInfo(file).symLinkTarget();
 }
+
+QString ScriptFileInfo::cleanPath(const QString &file) const
+{
+    return QDir().cleanPath(file);
+}
+
+QString ScriptFileInfo::fromNativeSeparators(const QString &file) const
+{
+    return QDir().fromNativeSeparators(file);
+}
+
+QString ScriptFileInfo::toNativeSeparators(const QString &file) const
+{
+    return QDir().fromNativeSeparators(file);
+}
+
+bool  ScriptFileInfo::cd(const QString &file)
+{
+    return QDir().cd(file);
```

Comment:
This is not a very useful binding, with `QDir::cd` just meant to change the directory the QDir instance is pointing to, but we're changing a temporary instance. The only use for this would to check if the directory exists, since that's what it returns, but we already have `exists` for this.

Reply:
sounds good.